packages/cli-auth/credentials-store.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'node:path';
+import * as fs from 'node:fs';
+import { homedir } from 'node:os';
+// NOTE: We are pinned on v5 because newer versions behave differently
+// when there are dots in the application name, eg. `com.vercel.cli` becomes `com.vercel`
+import XDGAppPaths from 'xdg-app-paths';
+import { z } from 'zod/mini';
+
+const Credentials = z.object({
+  /** An `access_token` obtained using the OAuth Device Authorization flow.  */
+  token: z.optional(z.string()),
+  /** A `refresh_token` obtained using the OAuth Device Authorization flow. */
+  refreshToken: z.optional(z.string().check(z.minLength(1))),
```

Comment:
Why have the minLength check on the `refreshToken` but not the `token`? I think we can be relaxed here and remove the check.
```suggestion
  /** An `access_token` obtained using the OAuth Device Authorization flow.  */
  token: z.optional(z.string()),
  /** A `refresh_token` obtained using the OAuth Device Authorization flow. */
  refreshToken: z.optional(z.string()),
```